R/nested.R

```diff
@@ -146,24 +150,25 @@ set_spaces <- function(spaces_after_prefix, force_one) {
 #' @importFrom purrr map2
 nest_parse_data <- function(pd_flat) {
   if (all(pd_flat$parent <= 0)) return(pd_flat)
+
+
+  pd_flat$internal <- with(pd_flat, (id %in% parent) | (parent <= 0))
   split <- pd_flat %>%
-    mutate_(internal = ~ (id %in% parent) | (parent <= 0)) %>%
-    nest_("data", names(pd_flat))
+    nest_("data", setdiff(names(pd_flat), "internal"))
```

Comment:
You could try `split_data <- split(pd_flat[...], pd_flat$internal)` and use `split_data` instead of `split$data` below.